jenkins-node/docker/UbuntuXenial.Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
       curl \
       gdebi-core \
+      psmisc \
```

Comment:
Slight OCD but can the packages remain alphabetically ordered?

Reply:
:+1: